benchmarks/common.py

```diff
@@ -1616,6 +1631,44 @@ def main(runner, original_dir=None):
         print_summary(output_filename)
 
 
+def log_operator_inputs(model, example_inputs, model_iter_fn, name, args):
+    output_split = args.output.split("/")
```

Comment:
Use [os.path](https://docs.python.org/3/library/os.path.html) for filename manipulation.